<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardDialog</class>
 <widget class="QDialog" name="KeyboardDialog">
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_0">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_1">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>w</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_2">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_3">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>r</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_4">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_5">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_6">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>u</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_7">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_8">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>o</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_9">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>p</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_10">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_11">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_12">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_13">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>f</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_14">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>g</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_15">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_16">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>j</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_17">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_18">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnShift">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="icon">
        <iconset resource="pixmaps.qrc">
         <normaloff>:/capslock_off.png</normaloff>
         <normalon>:/capslock_on.png</normalon>:/capslock_off.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>52</width>
         <height>52</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_19">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_20">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_21">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>c</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_22">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_23">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_24">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>n</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_25">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="icon">
        <iconset resource="pixmaps.qrc">
         <normaloff>:/backspace.svg</normaloff>:/backspace.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btnMod">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>88</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>123</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_26">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEnter">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>ENT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>88</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>CLOSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pixmaps.qrc"/>
 </resources>
 <connections/>
</ui>
